### PR TITLE
Implement Privet on Windows fixes #246

### LINKED ISSUE #246: Privet on Windows
Support privet on windows.

### PR DESCRIPTION
**Pull request by jay0lee**
First shot at Privet on Windows for local printing. Code needs a lot of
work but on my test machine, local printing seems to work for basic
scenarios.

This patch uses github.com/andrewtj/dnssd for mDNS which in turn relies
on Apple's Bonjour Service for Windows which must be installed:

https://support.apple.com/kb/dl999?locale=en_US

Apple does allow Bonjour Service to be distrubuted in some cases so we
may be able to wrap it into the MSI installer but for now it will need
to be installed before cloud-print-connector.

### CODE REVIEW

**Comment by agoode** [comment]
If github.com/andrewtj/dnssd is the right thing, we should use it to replace the custom bonjour code we have for Mac, unifying Mac and Windows bonjour support within this project.

**Comment by jay0lee** [comment]
Accidentally closed while squashing commits. Re-opening.

File: privet/windows.go [hunk 1]
```diff
@@ -9,25 +9,104 @@
 package privet
 
 import (
-	"errors"
+  "fmt"
```
Comment (agoode) [inline privet/windows.go]: Need to run `go fmt`.

File: privet/windows.go [hunk 2]
```diff
@@ -9,25 +9,104 @@
...
+  "sync"
+  "github.com/andrewtj/dnssd"
```
Comment (agoode) [inline privet/windows.go]: Is it feasible to move the Mac support to this library first?

Comment (jay0lee) [inline privet/windows.go]: Does it make sense to let the Windows implementation bake in first before switching Mac also?

Comment (agoode) [inline privet/windows.go]: I am happy to go in either order. My thought was that if we moved Mac over first (which works today), we would find potential issues that might cause problems on Windows, thus improving the chances Windows would work properly.

**Pull request closed without merging**